app/src/main/java/com/gdsctsec/smartt/ui/MainActivity.kt

```diff
@@ -39,11 +34,11 @@ class MainActivity : AppCompatActivity(){
 
     bottomNavigationView.setOnItemSelectedListener {
         when(it.itemId){
-                R.id.home_button-> {
+                R.id.home_button -> {
                     viewPager.currentItem=0
                     true
                 }
-            R.id.calender_button->{
+            R.id.calender_button ->{
```

Comment:
typo! refactor this id to `calendar_button`, always better to keep the code clean from typos/spelling mistakes even though it doesn't matter